eip-progpow.md

```diff
@@ -0,0 +1,115 @@
+---
+EIP <to be assigned> 
+Title: ProgPoW, a Programmatic Proof-of-Work
+Author: @RadixPi <radix.pi.314@gmail.com> | @IfDefElse <ifdefelse@protonmail.com>   
```

Comment:
Please either use "Real Name (@username)" or "Real Name <email>" format, and separate with commas.